src/dupininterface.cpp

```diff
@@ -0,0 +1,25 @@
+#include "dupin.h"
+#include <pybind11/eigen.h>
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+namespace py = pybind11;
+
+PYBIND11_MODULE(_dupin, m) {
+  py::class_<DynamicProgramming>(m, "DynamicProgramming")
+      .def(py::init<>())
+      .def_property("data", &DynamicProgramming::getDatum,
+                    &DynamicProgramming::setDatum)
+      .def_property("cost_matrix", &DynamicProgramming::getCostMatrix,
+                    &DynamicProgramming::setCostMatrix)
+      .def_property("num_bkps", &DynamicProgramming::get_num_bkps,
+                    &DynamicProgramming::set_num_bkps)
+      .def_property("num_timesteps", &DynamicProgramming::get_num_timesteps,
+                    &DynamicProgramming::set_num_timesteps)
+      .def_property("num_parameters", &DynamicProgramming::get_num_parameters,
+                    &DynamicProgramming::set_num_parameters)
+      .def("initialize_cost_matrix",
+           &DynamicProgramming::initialize_cost_matrix)
+      .def("return_breakpoints", &DynamicProgramming::return_breakpoints)
```

Comment:
The final C++ compute/fit/operator() method should return breakpoints.

Reply:
Forgot to push that, it will be with the next one in a few